librz/core/cfile.c

```diff
@@ -561,16 +529,14 @@ CURSOR cursor position(offset from curseek)
 	sdb_sync(config_sdb);
 	sdb_free(config_sdb);
 	rz_sys_setenv("RZ_CONFIG", config_sdb_path);
-
 	rz_sys_setenv("RZ_BIN_LANG", rz_config_get(core->config, "bin.lang"));
 	rz_sys_setenv("RZ_BIN_DEMANGLE", rz_config_get(core->config, "bin.demangle"));
 	rz_sys_setenv("RZ_ARCH", rz_config_get(core->config, "asm.arch"));
 	rz_sys_setenv("RZ_BITS", sdb_fmt("%" PFMT64u, rz_config_get_i(core->config, "asm.bits")));
-	rz_sys_setenv("RZ_COLOR", rz_config_get_i(core->config, "scr.color") ? "1" : "0");
-	rz_sys_setenv("RZ_DEBUG", rz_config_get_b(core->config, "cfg.debug") ? "1" : "0");
-	rz_sys_setenv("RZ_IOVA", rz_config_get_i(core->config, "io.va") ? "1" : "0");
+	rz_sys_setenv("RZ_COLOR", sdb_fmt("%" PFMT64u, rz_config_get_i(core->config, "scr.color")));
+	rz_sys_setenv("RZ_DEBUG", rz_config_get_b(core->config, "cfg.debug") ? "true" : "false");
```

Comment:
I suggest to just create some uber-simple RzUtil function to convert `bool` to string and use it here.

Reply:
tbh, i do not see any reason to have this. maybe a macro, but still, not really needed.

Reply:
In fact there is already: `rz_str_bool()`. Moreover, if you just use `rz_config_get(core->config, "cfg.debug")` it should return "true" or "false" string - it checks if the variable is boolean internally already, see `librz/config/config.c`.

Reply:
that code is old. i will revert it back to the original code since i dropped due the new usage of the RzSubprocess